docs/samples/kafka/README.md

```diff
@@ -0,0 +1,161 @@
+  --filename https://github.com/knative/eventing/releases/download/v0.10.0/release.yaml
+kubectl apply --filename https://github.com/knative/eventing/releases/download/v0.10.0/release.yaml
+kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/v0.10.2/release.yaml
+```
+Apply the `InferenceService` addressable cluster role
+```bash
+kubectl apply -f addressable-resolver.yaml
+```
+## Deploy Minio
+- If you do not have Minio setup in your cluster, you can run following command to install Minio test instance.
+```bash
+kubectl apply -f minio.yaml
+```
+
+- Install Minio client [mc](https://docs.min.io/docs/minio-client-complete-guide)
+```bash
+kubectl port-forward $(kubectl get pod --selector="app=minio" --output jsonpath='{.items[0].metadata.name}') 9000:9000
+mc config host add myminio http://127.0.0.1:9000 minio minio123
+```
+- Create buckets `mnist` for uploading images and `digit-[0-9]` for classification.
+```bash
+mc mb myminio/mnist
+```
+
+- Setup event notification to publish events to kafka.
+```bash
+mc event add myminio/mnist arn:minio:sqs:us-east-1:1:kafka --suffix .png
+```
+
+you should expect a notification event like following sent to kafka topic `mnist` after uploading an image in `mnist` bucket
+```json
+{
+   "EventType":"s3:ObjectCreated:Put",
+   "Key":"mnist/0.png",
+   "Records":[
+      {"eventVersion":"2.0",
+       "eventSource":"minio:s3",
+       "awsRegion":"",
+       "eventTime":"2019-11-17T19:08:08Z",
+       "eventName":"s3:ObjectCreated:Put",
+       "userIdentity":{"principalId":"minio"},
+       "requestParameters":{"sourceIPAddress":"127.0.0.1:37830"},
+       "responseElements":{"x-amz-request-id":"15D808BF706E0994",
+       "x-minio-origin-endpoint":"http://10.244.0.71:9000"},
+       "s3":{
+          "s3SchemaVersion":"1.0",
+          "configurationId":"Config",
+          "bucket":{
+               "name":"mnist",
+               "ownerIdentity":{"principalId":"minio"},
+               "arn":"arn:aws:s3:::mnist"},
+          "object":{"key":"0.png","size":324,"eTag":"ebed21f6f77b0a64673a3c96b0c623ba","contentType":"image/png","userMetadata":{"content-type":"image/png"},"versionId":"1","sequencer":"15D808BF706E0994"}},
+          "source":{"host":"","port":"","userAgent":""}}
+   ],
+   "level":"info",
+   "msg":"",
+   "time":"2019-11-17T19:08:08Z"
+}
+```
+
+## Train TF mnist model and save on S3
+Follow Kubeflow's [TF mnist example](https://github.com/kubeflow/examples/tree/master/mnist#using-s3) to train a TF mnist model and save on S3,
+change following S3 access settings, `modelDir` and `exportDir` as needed. If you already have a mnist model saved on S3 you can skip this step.
+```bash
+export S3_USE_HTTPS=0 #set to 0 for default minio installs
+export S3_ENDPOINT=minio-service:9000
+export AWS_ENDPOINT_URL=http://${S3_ENDPOINT}
+
+kustomize edit add configmap mnist-map-training --from-literal=S3_ENDPOINT=${S3_ENDPOINT}
```

Comment:
These kustomize commands did not work for me. Which version is required?
```
kustomize edit add configmap mnist-map-training --from-literal=S3_ENDPOINT=${S3_ENDPOINT}
Error: Missing kustomization file 'kustomization.yaml'.
```

Reply:
```
kustomize version
Version: {KustomizeVersion:3.1.0 GitCommit:95f3303493fdea243ae83b767978092396169baf BuildDate:2019-07-26T18:11:16Z GoOs:linux GoArch:amd64}
```

Maybe add required version to requirements at top

Reply:
added the requirement.